src/features/search/pages/modals/PriceModal/PriceModal.tsx

```diff
@@ -86,7 +86,9 @@ export const PriceModal: FunctionComponent<Props> = ({
 
   function search(values: PriceModalFormData) {
     const offerIsFree =
-      values.isOnlyFreeOffersSearch || (values.maxPrice === '0' && values.minPrice === '')
+      values.isOnlyFreeOffersSearch ||
+      (values.maxPrice === '0' && values.minPrice === '') ||
+      (values.maxPrice === '0' && values.minPrice === '0')
```

Comment:
Je ne connais pas la politique de qualité concernant ce genre de chose chez Pass mais je peux te proposer :

```suggestion
      values.isOnlyFreeOffersSearch ||
      (values.maxPrice === '0' && (values.minPrice === '' || values.minPrice === '0'))
```
pour un code plus court

Ou encore :
```suggestion
      values.isOnlyFreeOffersSearch ||
      (values.maxPrice === '0' && ['', '0'].includes(values.minPrice === ''))
```